python/spec/dependabot/python/update_checker/index_finder_spec.rb

```diff
@@ -309,5 +309,40 @@
         end
       end
     end
+
+    describe "#environment_variables" do
+      subject { finder.clean_check_and_remove_environment_variables(*arguments) }
```

Comment:
I've got no clue if this is a good idea or not, I copied it from here: https://zverok.github.io/blog/2017-11-01-rspec-method-call.html